ValveResourceFormat/IO/MapExtract.cs

```diff
@@ -822,9 +824,42 @@ private static ContentFile GeneratePhysicsTagMaterial(string surfaceName, IFileL
         materialContentFile.FileName = HammerMeshBuilder.proceduralPhysMaterialsPath + surfaceName + ".vmat";
         assetReferences.Add(materialContentFile.FileName);
 
+        GeneratePhysicsTagTexture(surfaceName, "D:/S2 tools");
+
         return materialContentFile;
     }
 
+    private static void GeneratePhysicsTagTexture(string surfaceName, string outputPath)
+    {
+        var text = "S2V AUTOPHYS " + surfaceName.ToUpper(CultureInfo.CurrentCulture);
+        int texSize = 64;
+        SKColor backgroundColor = SKColors.DarkBlue;
+
+        using (var bitmap = new SKBitmap(texSize, texSize))
+        {
+            using (SKCanvas canvas = new SKCanvas(bitmap))
+            {
+                canvas.Clear(backgroundColor);
+
+                using var textPaint = new SKPaint
+                {
+                    Color = SKColors.White,
+                    TextSize = 10,
+                    TextAlign = SKTextAlign.Center,
+                    IsAntialias = true,
+                    Typeface = SKTypeface.FromFamilyName("Arial"),
```

Comment:
This is a Windows font, so it'll be problematic on other systems (you can export with cli).